src/content/get-started/quick.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Get started with Flutter
+short-title: Get started
+description: Get started developing your first app with Flutter!
+toc: false
+show_breadcrumbs: false
+showBanner: false
+---
+
+Learn how to use VS Code to set up your Flutter development environment
+and test drive Flutter's developer experience.
+
+:::note What you'll achieve
+
+- Install the software prerequisites for Flutter.
+- Use VS Code to download and install Flutter.
+- Create a new Flutter app from a sample template.
+- Try out Flutter development features like stateful hot reload.
+
+:::
+
+## Choose your development platform
+
+The instructions on this page are configured to cover setting up Flutter on
+a **Windows**{:.selected-os-text} device.
+
+If you'd like to follow the instructions for a different OS,
+please select one of the following.
+
+{% osSelector %}
+
+## Download prerequisite software
+
+For the smoothest Flutter setup,
+first install the following tools.
+
+ 1. <h3>Install Xcode command-line tools</h3>
```

Comment:
This page looks awesome. I have a quick question. If we want to apply this number styling to other topics (flavors, in my case), would we be able to do this?

Reply:
Yes, you can use this already. The first line should be wrapped in `h3`, then add an extra new line and then `{: .steps}` after the list.

```md
 1. <h3>First item header</h3>
  
    First item contents.
   
 2. <h3>Second item header</h3>
 
    Second item contents.
    
{: .steps}